### ISSUE TITLE
KeyError and issue in filter_logits

### ISSUE THREAD

**Issue by nikhilsos** [legoodmanner/jukedrummer#3]
There's a key error while doing the inference from the saved pkl file (exp1) in my case,

Update:
I think there's a mismatch in key from that of saved model. Changed the keys, worked..
 also the filter logits function has to be changed: sorted_indices_to_remove  --> make boolean and sorted_indices_to_remove to tensor. For those using python 3.12 or > , 
install torch-fast-transformers and jukebox(source) as it is not mentioned in the requirements  and madmom 0.16.1 had to be source installed then pyproject toml had to be updated.

Hope it helps if someone else has same issue in the future.

**Comment by klimkin-vlad** [comment]
[nikhilsos](https://github.com/nikhilsos), Which version of Jukebox do you use?

**Comment by nikhilsos** [comment]
> [nikhilsos](https://github.com/nikhilsos), Which version of Jukebox do you use?

I built it from source after cloning the jukebox GitHub repository.

**Comment by klimkin-vlad** [comment]
Did you change any file of this (https://github.com/openai/jukebox) or current repository?

**Comment by nikhilsos** [comment]
> Did you change any file of this (https://github.com/openai/jukebox) or current repository?

Some minor tweaks here and there whenever some issue came..

**Comment by 3x10e8** [comment]
Thank you so much @nikhilsos for your suggestions, they were very helpful!

For the keyerror on loading model weights, I updated the key names as you suggested:
https://github.com/3x10e8/jukedrummer/blob/cb252f0939b6ff8f27c3d6e933c10f1af8c3677a/inference.py#L69-L80

For the dtype error in `sorted_indices_to_remove`, I wasn't able to get `bool` to work, but `torch.uint8` worked (although I get a copy warning for not copying the tensor correctly):
https://github.com/openai/jukebox/blob/cc22313ea940c11b52d52c423246f58f6ab86b9a/jukebox/transformer/ops.py#L139-L141

For requirements, I updated the requirements.txt on the fork. 
https://github.com/3x10e8/jukedrummer/blob/master/requirements.txt

Still wasn't able to get `madmom` to install from `pip install -r requirements.txt`, but installing it separately after installing the other requirements worked.

All of the other tweaks I made are summarized in this fork's commit log: https://github.com/3x10e8/jukedrummer/commit/cb252f0939b6ff8f27c3d6e933c10f1af8c3677a

If you could share yours or notice anything that could be done better, please let me know, thank you!
